chat-android/src/main/java/com/ably/chat/Typing.kt

```diff
@@ -163,82 +187,132 @@ internal class DefaultTyping(
     override suspend fun get(): Set<String> {
         logger.trace("DefaultTyping.get()")
         room.ensureAttached(logger) // CHA-T2d, CHA-T2c, CHA-T2g
-        return channelWrapper.presence.getCoroutine().map { it.clientId }.toSet()
+        return currentlyTypingMembers
     }
 
     override suspend fun start() {
         logger.trace("DefaultTyping.start()")
-
-        typingScope.launch {
-            // If the user is already typing, reset the timer
-            if (typingJob != null) {
-                logger.debug("DefaultTyping.start(); already typing, resetting timer")
-                typingJob?.cancel()
-                startTypingTimer()
-            } else {
-                startTypingTimer()
-                room.ensureAttached(logger) // CHA-T4a1, CHA-T4a3, CHA-T4a4
-                channelWrapper.presence.enterClientCoroutine(room.clientId)
+        typingScope.async {
+            room.ensureAttached(logger) // CHA-T4a1, CHA-T4a3, CHA-T4a4
+            // If the user is already typing, return
+            if (typingInProgressJob?.isActive == true) {
+                logger.warn("DefaultTyping.start(); already typing, so it's a no-op")
```

Comment:
Shouldn't be `warn`, should be `trace` or `info`

Reply:
will update to `info`

Reply:
Fixed https://github.com/ably/ably-chat-kotlin/pull/117/commits/00a0e7d3b66ac64a87b69cdfc872cd601f0efe04